### PR TITLE
Feature/window density model improvements

### PR DESCRIPTION
**Pull request by sayanchk**
This a merge request for a dev release for the improved window density model.

### CODE REVIEW

File: luminaire/exploration/data_exploration.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+from luminaire.model.model_utils import LADHolidays
```
Comment (shahsmit14) [inline luminaire/exploration/data_exploration.py:1]: Later we might want to think to rename LADHolidays to LADUSHolidays. Or how we users can pass region specific holidays.

File: luminaire/model/window_density.py [hunk 2]
```diff
@@ -437,18 +393,14 @@ def train(self, data, **kwargs):
...
+        (True, "2018-10-10 23:00:00", <luminaire.model.window_density.WindowDensityModel object at 0x7fd7c5a34e80>)
         """
         import numpy as np
         import pandas as pd
 
-        freq = self._params['freq']
-        min_num_train_windows = self.min_num_train_windows
-        max_num_train_windows = self.max_num_train_windows
-        ignore_window = self._params['ignore_window']
+        freq = pd.Timedelta(self._params['freq']) if self._params['freq'] not in ['S', 'T', '15T', 'H', 'D'] \
```
Comment (shahsmit14) [inline luminaire/model/window_density.py:401]: We should create a list variable for such time frequencies, to make it easier to update code in future.

**Review by shahsmit14 — CHANGES_REQUESTED** [review]
There are few binary files which are getting pushed as-well. We should remove them.

File: luminaire/exploration/data_exploration.py (hunk 1 above)
Comment (sayanchk) [reply luminaire/exploration/data_exploration.py:1]: This is a good suggestion. I'll create an issue

File: luminaire/model/window_density.py (hunk 2 above)
Comment (sayanchk) [reply luminaire/model/window_density.py:401]: The frequency can be arbitrary for this model. I just listed some typical ones to apply the corresponding settings.

**Review by shahsmit14 — APPROVED** [review]

**Comment by shahsmit14** [comment]
You will need to update the setup.py file to match the dev/pre-release version.

**Review by shahsmit14 — APPROVED** [review]

**Pull request merged**